contracts/governance/README.adoc

```diff
@@ -22,6 +22,8 @@ Votes modules determine the source of voting power, and sometimes quorum number.
 
 * {GovernorVotes}: Extracts voting weight from an {ERC20Votes} token.
 
+* {GovernorVotesERC721}: Extracts voting weight from an {ERC721Votes} token.
```

Comment:
We have GovernorVotes, so I guess this is not needed, is it ?

Reply:
You are right